dags/public-health/covid19/jhu-county-to-esri.py

```diff
@@ -264,6 +266,100 @@ def integrify(x):
     return df
 
 
+########### T2 and Helper Functions ###################################
+def subset_msa(df):
+    # 5 MSAs to plot: NYC, SF_SJ, SEA, DET, LA
+    df = df[
+        df.cbsatitle.str.contains("Los Angeles")
+        | df.cbsatitle.str.contains("New York")
+        | df.cbsatitle.str.contains("San Francisco")
+        | df.cbsatitle.str.contains("San Jose")
+        | df.cbsatitle.str.contains("Seattle")
+        | df.cbsatitle.str.contains("Detroit")
+    ]
+
+    def new_categories(row):
+        if ("San Francisco" in row.cbsatitle) or ("San Jose" in row.cbsatitle):
+            return "SF/SJ"
+        elif "Los Angeles" in row.cbsatitle:
+            return "LA/OC"
+        elif "New York City" in row.cbsatitle:
+            return "NYC"
+        elif "Seattle" in row.cbsatitle:
+            return "SEA"
+        elif "Detroit" in row.cbsatitle:
+            return "DET"
+
+    df = df.assign(msa=df.apply(new_categories, axis=1))
+
+    return df
+
+
+def update_msa_dataset(**kwargs):
+    """
+    Update MSA dataset
+    ref gh/aqueduct#199
+    takes the previous step data, aggegrates by MSA
+    replaces featurelayer.
+    """
+    # arcconnection = BaseHook.get_connection("arcgis")
+    # arcuser = arcconnection.login
+    # arcpassword = arcconnection.password
+    arcuser = "hunterowens"
+    arcpassword = "EDBQuESzo9j9"
```

Comment:
Oops!

Reply:
I took it out

Reply:
test now? I hopefully rebased succesfully?

Reply:
ugh nope